lib/faker/gov/it.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Faker.Gov.It do
+  @moduledoc """
+  Generating Italian fiscal codes
```

Comment:
I assume it could be more than just this one. I suggest we generalise it.

Reply:
The fiscal id in Italy is basically an SSN, as you can read  [here](https://en.wikipedia.org/wiki/Italian_fiscal_code_card). It's fine if we put the module under `Faker.Gov.It.ssn/0`, and we can discuss about having an aliased `Faker.Gov.It.fiscal_id/0` if it's useful

Reply:
What I meant is - I expect it eventually have tax numbers for example, or any additional generators. Currently module description tight to 1 function.

Reply:
Oh, now I get it (: 
I blatantly copied it from `Faked.Gov.Us`, so...